src/modules/actions/user.py

```diff
@@ -137,11 +137,56 @@ def install(self, pkgplan, orig, retry=False):
                         return
 
                 username = self.attrs["username"]
+                #sys.stderr.write("[D] Processing user " + username + "\n")
```

Comment:
Line can be deleted (Code Cleanup)

Reply:
The commented traces are actually in a separate commit that may be omitted. For now I kept them so that an installed `pkg` package can be more easily troubleshooted on a system, if this new logic causes problems (or is suspected to) by uncommenting stuff. An alternative may be to tap into environment variables, some parts of `pkg` do that already to tweak behaviors.

Reply:
We cannot pick out individual commits in GitHub like in the CLI and I would like to only pull via Web not CLI. Uncommenting is only an accessible to the developer and not any user. If you would like to have a mechanism to debug this in running systems, it is better to Guard that behind env Variables or use debugger assisted debugging rather than printf debugging.

Reply:
Makes sense; I'll look at that after my current quest then. Do you know OTOH if there already is some debugging/tracing envvar in pkg, to not define another?

Reply:
I am not aware of any.

Reply:
Seems there is a gazillion items checked in `os.environ` but most seem tailored to their occasion. So adding another would follow established precedent...

Reply:
Updated with envvar for tracing, another envvar to just report the collision and move on like current code would do, and fixed indentation (8 spaces).

Reply:
Awesome LGTM